packages/pyright-internal/src/parser/unicode.ts

```diff
@@ -291,6 +291,8 @@ export const unicodeLu: UnicodeRangeTable = [
     4295,
     4301,
     [5024, 5109],
+    [7312, 7354],
```

Comment:
@erictraut, I'd like to make two changes to the file format if you're OK with it:
1. Change character codes from decimal to hex.
2. Add comments on surrogate table lines with the UCS character codes in hex. Or even the actual characters?

This would save a lot of numeric translation hassle when a human tries to understand what characters are in here vs. not. And should be an easy change to the generator.

Thinking I should submit this as a separate change though.